CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ### Changes
 - Add handling of project limits [#1456](https://github.com/neptune-ai/neptune-client/pull/1456)
+- Language and style improvements [#1465](https://github.com/neptune-ai/neptune-client/pull/1465)
```

Comment:
Ensure that this will be a part of the latest version (`1.7.0`). Probably rebase is required.